cli/dcoscli/marathon/main.py

```diff
@@ -94,6 +96,8 @@
                                 stdin.
 
     <task-id>                   The task id
+
+    <scale-factor>              The value of scale-factor for a group
```

Comment:
`The value of scale-factor for a group` -> `The factor to scale an application group by.`